doc/using_the_api.md

```diff
@@ -222,6 +222,19 @@ All endpoints are prefixed with `/api/`
     ]
 ```
 
+#### ENDPOINT: USER INVITE
+
+| method | route           | action                 |
+| :----- | :-------------- | :--------------------- |
+| POST   | `/user/invite`  | Invite a user          |
+
+##### Response (200)
+
+```javascript
+    {
+        "message": "User Invited"
+    }
+```
```

Comment:
In addition to the response data, I think it would be useful to include a section showing some example request data.
Let's go ahead and add that with the following fields:
- `firstName` (string)
- `lastName` (string)
- `phone` (string)
- `email` (string)`
- `role` (int)

Reply:
Okay. I'll look for some sample request data and add that.

Reply:
Awesome, thanks!